cmd/nodecmd/create.go

```diff
@@ -136,7 +139,7 @@ func createNodes(_ *cobra.Command, args []string) error {
 	gcpCredentialFilepath := ""
 	if cloudService == constants.AWSCloudService {
 		// Get AWS Credential, region and AMI
-		ec2Svc, region, ami, err := getAWSCloudConfig(cmdLineRegion, authorizeAccess)
+		ec2Svc, region, ami, err := getAWSCloudConfig(awsProfile, cmdLineRegion, authorizeAccess)
```

Comment:
also make createAWSInstances to receive awsProfile, so there is no more global var assumption on create_aws.go

Reply:
done